src/components/HeaderBreadcrumbs/HeaderBreadcrumbsItem.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../../styles/mixins';
+@import '../../../styles/variables';
+
+$block: '.#{$ns}header-breadcrumbs-item';
+
+#{$block} {
+    &__item {
+        display: inline-block;
+    }
+
+    &__text {
+        @include text-size(body-2);
+        @include link();
+
+        color: var(--yc-color-text-secondary);
+
+        &:after {
+            content: '\a0/ ';
+            margin: 0 8px 0 6px;
```

Comment:
use margin form $indent variables

Reply:
I had looked carefully on this code and figured out - there is overcoding) we do not need to create new Components with analytics handlers in the PR. At least not now